docs/api-ref.md

```diff
@@ -748,6 +748,41 @@ An updated `ConnectionItem` for the data source.
 See the `update_connection.py` sample in the Samples directory.
 
 
+<br>
+<br>
+
+#### datasource.update_data
+
+```py
+datasource.update_date(datasource_or_connection_item, *, request_id, actions, payload)
+```
+
+Updates the data contained within a published live-to-Hyper datasource.
+
+REST API: [Update Data in Published Live-to-Hyper Data Sources](https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_how_to_update_data_to_hyper.htm)
+
+
+**Parameters**
+
+Name   |  Description
+ :--- | : ---
+`datasource_or_connection_item` | Either a `DataSourceItem` or a `ConnectionItem`. If the datasource only contains a single connections, the `DataSourceItem` is sufficient to identify which data should be updated. Otherwise, for datasources with multiple connections, a `ConnectionItem` must be provided.
+`request_id` | used as an idempotency key
```

Comment:
Can you add a bit more explanation what this means and what to use if you don't care about this?